.eslintrc.js

```diff
@@ -159,7 +159,7 @@ module.exports = {
       },
     },
     {
-      files: ["*.tests.ts", "*.tests.tsx"],
+      files: ["*.tests.ts", "*.tests.tsx", "src/app/system/navigation/RouterLink.tsx"],
```

Comment:
```suggestion
      files: ["*.tests.ts", "*.tests.tsx"],
```

I prefer if we keep this as it and add 
`// eslint-disable-next-line no-restricted-imports` to `RouterLink`

```typescript
// eslint-disable-next-line no-restricted-imports
import { navigate } from "app/system/navigation/navigate"
```